support CSS variables using default values in react native (#3392)
BuilderIO/builder: .changeset/empty-drinks-nail.md

---

> "@builder.io/packages": patch

Reviewer: this is wrong and should list each SDK

> "@builder.io/packages": patch
> "@builder.io/sdks": patch

Reviewer: this is wrong and snould list 
```suggestion
"@builder.io/sdk-react-native": patch
```

---

> Support CSS variables that use default values in React Native

Reviewer: ```suggestion
Fix: Add support for CSS variables that use default values.
```
